### PR TITLE
Add validation to isActive param query and return boolean correctly

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/users/usersService.js [hunk 1]
```diff
@@ -40,10 +40,14 @@ class usersService {
     const search = query.map((criteria) => {
       if (regExpId.test(args[criteria])) {
         return { [criteria]: parseInt(args[criteria]) };
-      } else {
+      } else if (criteria === 'firstName') {
```
Comment (hyfi06) [inline components/users/usersService.js:43]: only "firstName"?

File: components/users/usersService.js [hunk 2]
```diff
@@ -40,10 +40,14 @@ class usersService {
...
         return {
           [criteria]: { $regex: new RegExp(`.*${args[criteria]}.*`, 'i') },
         };
+      } else if (criteria === 'isActive' && args[criteria] == 'true') {
+        return { [criteria]: true };
+      } else if (criteria === 'isActive' && args[criteria] == 'false') {
+        return { [criteria]: false };
```
Comment (hyfi06) [inline components/users/usersService.js:50]:
```
else if (criteria === 'isActive') {
        return { [criteria]: args[criteria] == 'true' };
}
```
`args[criteria] == 'true' ` is boolean.

File: components/users/usersService.js [hunk 3]
```diff
@@ -40,10 +40,14 @@ class usersService {
     const search = query.map((criteria) => {
       if (regExpId.test(args[criteria])) {
         return { [criteria]: parseInt(args[criteria]) };
-      } else {
+      } else if (criteria === 'firstName') {
         return {
           [criteria]: { $regex: new RegExp(`.*${args[criteria]}.*`, 'i') },
         };
+      } else if (criteria === 'isActive' && args[criteria] == 'true') {
```
Comment (hyfi06) [inline components/users/usersService.js:47]: check this first, before numbers.

**Comment by ealexisaraujo** [comment]
@hyfi06 Could you please check this commit

File: components/users/usersService.js (hunk 2 above)
Comment (hyfi06) [inline components/users/usersService.js]: Check this part, only filter by firstName, isActive, and number fields

File: components/users/usersService.js [hunk 4]
```diff
@@ -40,10 +40,14 @@ class usersService {
     const search = query.map((criteria) => {
```
Comment (hyfi06) [inline components/users/usersService.js]: `name` isn't in database

**Pull request merged**